poetry.lock

```diff
@@ -5272,13 +5394,13 @@ files = [
 
 [[package]]
 name = "requests"
-version = "2.31.0"
+version = "2.32.1"
```

Comment:
This has caused integration tests to fail... 
see https://github.com/docker/docker-py/issues/3256

I'll revert this change

Reply:
Rule of thumb:

Do

```bash
poetry lock --no-update
```

Don't do

```bash
poetry lock
```

Reply:
My bad, I should try `poetry lock --no-update`